src/main/webapp/app/components/calendarButton/UsageAnalysisCalendarButton.tsx

```diff
@@ -0,0 +1,132 @@
+  fromDate: (newDate: string) => void;
+  toDate: (newDate: string) => void;
+  filterToggled: (filterActive: boolean) => void;
+};
+
+export const UsageAnalysisCalendarButton: React.FunctionComponent<CalendarButtonProps> = props => {
+  function handleFromDayChange(day: any) {
+    props.fromDate(day);
+    props.filterToggled(true);
+    if (props.currentToDate == null || props.currentToDate === 'Select Date') {
+      props.toDate(day);
+    }
+  }
+
+  function handleToDayChange(day: any) {
+    props.toDate(day);
+    props.filterToggled(true);
+  }
+
+  function handleResetDays() {
+    props.fromDate('Select Date');
+    props.toDate('Select Date');
+    props.filterToggled(false);
+  }
+
+  return (
+    <DefaultTooltip placement={'top'} overlay={'View Specific Date Range'}>
+      <DropdownButton
+        as={InputGroup.Append}
+        title={<i className={classnames('fa fa-calendar')}></i>}
+        id="time-select-dropdown"
+        className="ml-3"
+      >
+        <Container>
+          <div>
+            <Row>
+              <Col>From:</Col>
+            </Row>
+          </div>
+          <DayPickerInput
+            value={
+              props.currentFromDate === undefined ||
+              props.currentFromDate === 'Select Date'
+                ? 'Select Date'
+                : getMomentInstance(props.currentFromDate).format(
+                    APP_LOCAL_DATE_FORMAT
+                  )
+            }
+            onDayChange={handleFromDayChange}
+            inputProps={{
+              style: {
+                textAlign: 'center',
+                border: 'none',
+              },
+            }}
+            style={{ margin: '0 auto' }}
+            formatDate={MomentLocaleUtils.formatDate}
+            parseDate={MomentLocaleUtils.parseDate}
+          />
+          <div>
+            <Row>
+              <Col>To:</Col>
+            </Row>
+          </div>
+          <DayPickerInput
+            value={
+              props.currentToDate === undefined ||
+              props.currentFromDate === 'Select Date'
+                ? 'Select Date'
+                : getMomentInstance(props.currentToDate).format(
+                    APP_LOCAL_DATE_FORMAT
+                  )
+            }
+            onDayChange={handleToDayChange}
+            inputProps={{
+              style: {
+                textAlign: 'center',
+                border: 'none',
+              },
+            }}
+            dayPickerProps={{
+              initialMonth:
+                props.currentToDate === undefined
+                  ? moment().toDate()
+                  : getMomentInstance(props.currentToDate).toDate(),
+            }}
+            style={{ margin: '0 auto' }}
+            formatDate={MomentLocaleUtils.formatDate}
+            parseDate={MomentLocaleUtils.parseDate}
+          />
+          <Row className={'mt-1'}>
+            <Button
+              size={'sm'}
+              style={{ margin: '0 auto', zIndex: 0 }}
```

Comment:
What happens if you don't specify zIndex?

Reply:
zIndex pushes the reset button all the way to the back so it doesn't appear above the calendar like so:

<img width="284" alt="Screen Shot 2022-07-23 at 10 03 47 AM" src="https://user-images.githubusercontent.com/71607240/180610710-3a18890d-c99d-46e8-8bbf-b1ae8b4408f2.png">